flows/local.ts

```diff
@@ -77,4 +77,23 @@ export const localFlows: Flow[] = [
     ctaUrl: "https://discord.gg/sismo",
     congratulationTexts: ["Provide feedback on Discord", "Join Snapshot Space"],
   },
+  {
+    path: "top-100-ens",
+    attester: hydraS1LocalAttester.name,
+    network: Network.Local,
+    attesterType: "hydra-s1",
+    badgesCollection: hydraS1LocalBadges,
+    badgesInternalCollectionsIds: [3],
+    title: "Top 100 ENS",
+    logoUrl: null,
+    subtitle: "Prove that you are across the top 100 ENS names by number of followers on Twitter",
+    onboardingDescription:
+      "This ZK Badge is an attestation that you are part of the top 100 ENS accounts by followers on Twitter. It can be used across different platform as a proof-of-reputation tool",
+    ctaLabel: "Join the anonymous conversation on Discord",
```

Comment:
"cta" means Call to action. It's the parameter that let you configure the button at the end of the flow. 
As here you don't have an anonymous conversation on our discord you should use
```
    ctaLabel: "See your badges",
    ctaUrl: "",
```